docs/installation.rst

```diff
@@ -215,6 +215,15 @@ These configuration you can use with django-is-core in your django settings file
     def perform_action_before_export_task(request, queryset, filename, **kwargs):
         logger.info(f"Exporting result of the query ({queryset.query}) into the file '{filename}'.")
 
+.. attribute:: IS_CORE_BACKGROUND_EXPORT_TASK_HANDLED_EXCEPTIONS
```

Comment:
```suggestion
.. attribute:: IS_CORE_BACKGROUND_EXPORT_TASK_CALLBACK_HANDLED_EXCEPTIONS
```
I would name it, as I suggested, `IS_CORE_BACKGROUND_EXPORT_TASK_CALLBACK_HANDLED_EXCEPTIONS` to indicate, that only exceptions, that arise from this callback call are relevant.